### DISCUSSION TITLE
SMPlayer Not Playing Youtube

### DISCUSSION THREAD

**Discussion by Ataru320** [smplayer-dev/smplayer#1005] [General]
Ever since Youtube has been way more problematic with adblockers and forcing people to watch them, I've been using SMPlayer to watch my videos without any real issue.  It's been fine for a while...but now there is an issue.  All of a sudden, whenever I put a Youtube link into my player, it just...doesn't work.  It appears to buffer and...nothing, no video, no nothing.  I upgraded my SMPlayer and its still not playing anything.  And while older links still work, it seems to so far only affect videos posted in the last few days, as if they're trying to change something to make it way harder.

Is there a workaround or something I can do?  because I don't really want to deal with Google/Youtube and their methods when I just want to watch my videos and give support with thumbs to support the algorhythm for those I do like on the format.

**Comment by Yucie** [comment]
In options, try to untick use adaptiv stream (resolution up to 4K). LIke this
![test33](https://github.com/user-attachments/assets/28eb1b20-09ce-440b-9631-8aeda7a567de)

Then use smplayer to watch normal video. For example:

https://www.youtube.com/watch?v=CkhrrqbIU2E

![test34](https://github.com/user-attachments/assets/a3549ed5-2297-47f8-ad17-7a843b7e29dc)


It can play normal video but watch boardcasting video will be sound delay at least 5 seconds.

If you set use adaptive stream and watch normal video. I think you will like this
![347896752-9e9c7568-7567-4632-9ad5-f165d824e567](https://github.com/user-attachments/assets/bfebe3d8-ef3c-403b-81a8-cc017b72b363)

This problem occur since last week. If you set use adaptive stream. Try turn it off.
If you want to watch boardcasting video, you need turn it on.

**Comment by Ataru320** [comment]
I made the adjustments; but when I put a Youtube link in, it's still doing the same thing: it tries and nothing happens.  It refuses to play anything off Youtube with the adjustments you advised.

**Reply by Yucie** [reply]
Did you play the link without restart smplayer?
If you change the config in smplayer, you need restart smplayer.

Or you paste the link here that let me try.

**Reply by Ataru320** [reply]
I reset it and tried.  Nothing.  Still dead.

**Reply by Yucie** [reply]
Please give me a link to try it is working or not.

**Reply by Ataru320** [reply]
https://www.youtube.com/watch?v=M9AcqMzI_6Q

**Reply by Yucie** [reply]
> https://www.youtube.com/watch?v=M9AcqMzI_6Q

Working 
![test3333](https://github.com/user-attachments/assets/5cf5d761-8e1e-49d4-850d-b58d97559ba4)

Try using same version smplayer and setting with my setting

Version: 24.5.0 (revision 10277) (64-bit)
Portable Edition
Using Qt 5.15.2 (compiled with Qt 5.15.2)
Using MPV v0.37.0-522-ga56d8ff1

[smplayer.zip](https://github.com/user-attachments/files/16187966/smplayer.zip)

**Reply by Ataru320** [reply]
I copied the file and put it in the same place as the SMPlayer app as advised.  I tried to open the link...nothing.

**Reply by Yucie** [reply]
> I copied the file and put it in the same place as the SMPlayer app as advised. I tried to open the link...nothing.

1. Check the smplayer folder premission -> "Full Control"
2. Set the smplayer folder with "white list" to M$ window defender (MS window defender will sometime ban yt-dlp.exe when startup, i dont know why)

**Reply by Ataru320** [reply]
I did everything you told me to; I whilisted it in my antivirus (I don't have Microsoft Defender since that is blocked off by Microsoft 365 advertisements and just had to do it in Bitdefender).

Nothing.

**Reply by Yucie** [reply]
Can i ask you about your OS version and network?

As i know, adgurad home (adveristment block software) can install in some router.
Do you block some scripts by the router side?

Did your OS version was too old?

Or you check some network activity on Chrome (set adblock off) to see what items was blocked by network.

Moreover, you can try to open mpv.exe on smplayer folder, then drop youtube link on it.
Is it work?

https://github.com/user-attachments/assets/c9509673-fae7-44c3-a8a7-f0d9fbfe5860

P.S Try using yt-dlp.exe to download youtube video. If can't, i think it is yt-dlp.exe problem rather than smplayer.

**Reply by Ataru320** [reply]
I gave up and will just download them by another means.  I will not play Youtube's game and will just find my own way.
